catalog/ngis-cat/projects_ngis-cat_assets_DEA_NIDEM.jsonnet

```diff
@@ -0,0 +1,179 @@
+local id = 'projects/ngis-cat/assets/DEA/NIDEM';
+local subdir = 'ngis-cat';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local catalog_subdir_url = ee_const.catalog_base + subdir + '/';
+local parent_url = catalog_subdir_url + 'catalog.json';
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'National Intertidal Digital Elevation Model 25m 1.0.0',
+  'gee:type': ee_const.gee_type.image,
+  
+  description: |||
+    The National Intertidal Digital Elevation Model (NIDEM; Bishop-Taylor et al. 2018, 2019) is a continental-scale elevation dataset for Australia's exposed intertidal zone. 
+    NIDEM provides the first three-dimensional representation of Australia's intertidal sandy beaches and shores, tidal flats and rocky shores and reefs at 25 m spatial resolution, 
+    addressing a key gap between the availability of sub-tidal bathymetry and terrestrial elevation data. NIDEM was generated by combining global tidal modelling 
+    with a 30-year time series archive of spatially and spectrally calibrated Landsat satellite data managed within the Digital Earth Australia (DEA) platform. 
+    NIDEM complements existing intertidal extent products, and provides data to support a new suite of use cases that require a more detailed understanding 
+    of the three-dimensional topography of the intertidal zone, such as hydrodynamic modelling, coastal risk management and ecological habitat mapping.
+    The product: https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics
+    The definition of the four bands, including NIDEM, NIDEM_mask, NIDEM_uncertainty, and NIDEM_unfiltered please visit https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#details
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'australia',
+    'dea_australia',
+    'dem',
+    'nidem',
+  ],
+  providers: [
+    ee.producer_provider('Digital Earth Australia', 'https://cmi.ga.gov.au/data-products/dea/325/dea-intertidal-elevation-landsat#basics'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+
+  extent: ee.extent(108.81, -44.41, 157.82, -9.13, '1986-08-16T00:00:00Z', '2017-07-31T23:59:59Z'),
+
+  // Summaries contain additional information specific to the dataset type.
+  summaries: {
+
+    'gee:schema': [
+      {
+        name: 'dea_national_intertidal_digital_elevation_model',

```

Comment:
This is a field on the ee.Image?  This looks like you were going for the description about the actual raster band, which goes in `eo:bands`. However, this is just the text from the overall description. So I think you need to remove the schema.

Do you have the data loaded someplace where I can check out the image?

Reply:
Are you referring to the whole gee:schema section? Each band is described in `eo:bands`. So yes, I think this section can be removed.
The data is made available in my GEE Assets: https://code.earthengine.google.com/?asset=projects/ngis-cat/assets/DEA/NIDEM